setup.py

```diff
@@ -41,7 +41,7 @@
         'future>=0.16.0',
         'h5py>=2.7.1',
         'matplotlib>=1.3.1',
-        'numpy>=1.8.0',
+        'numpy>=1.8.0,<1.16',
```

Comment:
why?

Reply:
Cf updated summary.

Reply:
wish it didn't have to be that way, but, lgtm

Reply:
Hopefully it should be for a short amount of time...